libs/BaseKonnector.js

```diff
@@ -30,23 +30,28 @@ module.exports = class baseKonnector {
       this.accountId = cozyFields.account
```

Comment:
Not about my PR here by shouldn't we suppose to get the account ID directly from `account._id` here? It seems less error prone to me to use the last account fetched IMHO